main.cpp

```diff
@@ -73,7 +24,22 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
-int main() {
+//new function added that gives all color
+//code with the pair numbers
+
+void printAllCodesAndColors()
+{
+    for (int pairNumber = 1; pairNumber < 26; pairNumber++) {
```

Comment:
Do not hardcode the loop limit, since it is easy to forget updating this when we add a color.